mobile/native/src/unfunded_orders.rs

```diff
@@ -0,0 +1,123 @@
+use crate::event;
+use crate::event::EventInternal;
+use crate::event::FundingChannelTask;
+use crate::state;
+use crate::trade::order;
+use crate::trade::order::api::NewOrder;
+use anyhow::Result;
+use bitcoin::Address;
+use bitcoin::Amount;
+use futures::FutureExt;
+use std::time::Duration;
+use xxi_node::commons::ChannelOpeningParams;
+
+pub(crate) async fn submit_unfunded_channel_opening_order(
```

Comment:
🙃 Every channel that is not yet open is unfunded. We could go for `submit_unfunded_wallet_channel_opening_order` or `submit_channel_opening_order_without_utxos`.